lib/streamlit/elements/chat.py

```diff
@@ -132,6 +133,9 @@ def chat_message(
             image file; URL to fetch the image from; array of shape (w,h) or (w,h,1)
             for a monochrome image, (w,h,3) for a color image, or (w,h,4) for an RGBA image.
 
+            If None (default), uses default icons if participant is “user” or
+            “assistant”, or the first letter of participant.
```

Comment:
Nit: Feel like the following makes more sense (if I'm understanding correctly):
```
If None (default), uses default icons if participant is “user” or
            “assistant”, otherwise the first letter of participant name.
```

Reply:
I updated the docstring. We might want to introduce an additional `name` parameter later, so I used `participant value` here for now.